style/style.css

```diff
@@ -1,3 +1,576 @@
-#test {
-  color: red;
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+}
+
+.logBar {
+  display: none;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  z-index: 3;
+  box-shadow: 0 20px 20px rgba(0, 0, 0, 0.6);
+}
```

Comment:
- [ ] I suggest removing the `shadow` from under the header navigation bar. 

First because it is the requirement to stick as closely as possible to the design guideline. 

Second we use shadows if you put either type or an` image` over a `background` and there's` not enough contrast` to see the edges of the image that's on top, then add a subtle drop shadow `to help define the edges`. But if there's plenty of contrast already, don't add a drop shadow.

<img width="1152" alt="Screenshot 2023-03-17 at 11 49 50 PM" src="https://user-images.githubusercontent.com/44798044/225992754-156774dd-1c4e-4d6b-b3d5-e11605815004.png">